frontend/src/app/(pages)/profile/page.tsx

```diff
@@ -1,9 +1,69 @@
+"use client"
 import ProfileComponent from "@/components/profile/Profile"
+import { UserService } from "@/helpers/user/user_api_wrappers";
+import User from "@/types/user";
+import { Metadata } from "next"
+import { FormEvent, useEffect, useState } from "react";
+import { toast } from "react-toastify";
 
-export default function ProfilePage() {
+export const metadata: Metadata = {
```

Comment:
This might be just for me. But I am more comfortable to put the `metadata` declaration in `layout.tsx` instead of `page.tsx`. If you agree with me, do you mind creating a new `layout.tsx` file in `/profile` path and move `metadata` there?